src/rhelocator/api/schema/health.yml

```diff
@@ -0,0 +1,24 @@
+---
+definitions:
+  type: object
+  properties:
+    message:
+      type: string
+
+responses:
+  200:
+    description:
+    schema:
+      $ref: '#/definitions/images'
```

Comment:
There is no '#/definitions/images' in your yml file anymore! You can simply shorten that ref to '#/definitions' and it will work though. The $ref is just injecting the reference object (in this case definitions/images) at this specific point of the yml structure so you can have less code repitition

Reply:
oh ups - haha :smile: makes sense, thank you :smiley: